R/derive_var_extreme_date.R

```diff
@@ -109,44 +109,45 @@ derive_var_lstalvdt <- function(dataset,
 #' @examples
 #' library(dplyr, warn.conflicts = FALSE)
 #' library(admiraltest)
-#' data("dm")
-#' data("ae")
-#' data("lb")
-#' data("adsl")
+#' data("admiral_dm")
+#' data("admiral_ae")
+#' data("admiral_lb")
+#' data("admiral_adsl")
 #'
 #' # derive last known alive datetime (LSTALVDTM)
 #' ae_start <- date_source(
-#'   dataset_name = "ae",
+#'   dataset_name = "admiral_ae",
```

Comment:
The `dataset_name` should still just be `"ae"`. However, the list you pass as the `source_datasets` argument below should be `list(ae = admiral_ae)`.

Reply:
Same for all other dataset names.